articles/tech-resources/custom-bounce-domain.md

```diff
@@ -23,13 +29,16 @@ Once you have configured your DNS settings, register and verify the domain with
 If you prefer to use the SparkPost UI, you can create multiple bounce domains or single bounce domain that will be used for all of your outgoing mail.
 
 1. Select Account and then [Sending Domains](https://app.sparkpost.com/account/sending-domains).
-1. Click the New Domain button to add a sending domain.
-1. Add the displayed CNAME record to your sending domain.
-1. Select the "verify DNS records" orange text in the CNAME record box.
-1. Make sure the domain is verified by the green check mark.
+1. Click the New Domain button to add a sending domain. Again, please add a subdomain to avoid any issues with your root domain.
+1. Navigate to the DNS for your root domain and add the displayed CNAME record.
```

Comment:
This is confusing. You just said use the subdomain, but then this step says navigate to the DNS for your root domain and add the CNAME. You add the CNAME to the subdomain in the DNS.